exonum/src/sandbox/consensus/basic.rs

```diff
@@ -76,6 +76,19 @@ fn test_reach_one_height() {
     sandbox.assert_state(Height(2), Round(1));
 }
 
+/// idea of the test is to reach one height in the network with single validator
+#[test]
+fn test_one_validator() {
+    let sandbox = sandbox::timestamping_sandbox_builder()
+        .with_logger()
+        .with_validators(1)
```

Comment:
Isn't `with_validators(1)` default behavior?

Reply:
Nope.

Reply:
Hmm... For `TestKitBuilder` it's default, so as for me it's confusing. We have two structures with the same method but for one `with_validator(1)` is default, and for one it is not. I'm not sure if is is OK, but I think it is at least discussible.

Reply:
I'm sure it's ok.

Reply:
Ok then.